### PR TITLE
[libffi] Add support for CMake config

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kou** [comment]
Should we add `unofficial` prefix?

**Comment by cbezault** [comment]
@ras0219-msft, what's our current stance on the unofficial prefix?

File: ports/libffi/CMakeLists.txt [hunk 1]
```diff
@@ -122,10 +122,23 @@ endif()
 add_library(libffi ${FFI_SOURCES})
 
 install(TARGETS libffi
+    EXPORT ${PROJECT_NAME}Targets
     RUNTIME DESTINATION bin
     ARCHIVE DESTINATION lib
     LIBRARY DESTINATION lib)
 
+if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
```
Comment (vicroms) [inline ports/libffi/CMakeLists.txt]: Remove the conditional and add a call to `vcpkg_fixup_cmake_targets()` in the portfile to consolidate the CMake files.

Right now, the output is missing `libffiTargets-debug.cmake`.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/libffi/CMakeLists.txt (hunk 1 above)
Comment (kou) [reply ports/libffi/CMakeLists.txt]: Done.

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**